src/docs/arch-cursor.dox

```diff
@@ -429,21 +429,22 @@ reach a leaf page. Once we found a leaf page, we go into different conditions to
 random visible disk record from the page using WiredTiger's RNG.
 3. Second condition: Loop through all insert lists, and find one that is large (1000 or more entries), once found attempt
 to find a random insert record from the page.
-4. Third condition: If page has small amount of disk records (200 or less entries) or the leaf page is clean, attempt to
+4. Third condition: If the page has small amount of disk records (200 or less entries) or the leaf page is clean, attempt to
 find a random visible disk record from the page using WiredTiger's RNG.
 5. If all conditions fail, compute a random number between 0 - 250 and perform cursor->next or cursor->prev a random
-number of times between 0 and 250. Finally return the visible record that the cursor stopped on. Note: This can be on
-different leaf page than our original page that we descended from.
+number of times between 0 and 250. The cursor->next and cursor prev function makes sure that 
```

Comment:
If you use `cursor->next`, you should use `cursor->prev` as well instead of "cursor prev"

Reply:
Removed the occurences of cursor->next and cursor->prev.